Sources/App/Models/Package.swift

```diff
@@ -45,9 +45,6 @@ final class Package: Model, Content {
     @OptionalEnum(key: "processing_stage")
     var processingStage: ProcessingStage?
 
-    @Field(key: "last_commit_at")  // TODO: shouldn't this rather live in Repository? Is it needed at all?
-    var lastCommitAt: Date?
```

Comment:
I've moved this over to `Repository` to group all the git metadata together. It doesn't really make a difference with respect to lookups as we're already joining `Package` and `Repository`.